pkg/serviceaccount/serviceaccounts_controller.go

```diff
@@ -42,23 +43,27 @@ func nameIndexFunc(obj interface{}) (string, error) {
 }
 
 type ServiceAccountControllerOptions struct {
```

Comment:
optional: Personally, since this is the serviceaccount package, I'd just call this ControllerOptions.